examples/nodejs-simple/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:stretch
```

Comment:
I don't have a strong opinion, but did you consider

`FROM node:11` ?

Reply:
I started with `FROM node:10` (since that is the current LTS release) and using two containers - one to build and one to run - but then realized that with node there isn't really much of a building step. Like python, it's pretty much just drop the files into the directory and say go. 

The reason I chose this is it was the same as what is used by the nodejs sdk build image, so it was easy to copy/paste the installation bits for curl / node and it gives you a resulting image with a shell. 

I think there might be `node:10-alpine` which could work too if you'd rather have a slimmer base image.